geode-core/src/main/java/org/apache/geode/management/internal/configuration/converters/GatewayReceiverConverter.java

```diff
@@ -30,6 +37,26 @@ protected GatewayReceiver fromNonNullXmlObject(GatewayReceiverConfig xmlObject)
     receiver.setMaximumTimeBetweenPings(stringToInt(xmlObject.getMaximumTimeBetweenPings()));
     receiver.setSocketBufferSize(stringToInt(xmlObject.getSocketBufferSize()));
     receiver.setStartPort(stringToInt(xmlObject.getStartPort()));
+    List<ClassName> configFilters = new ArrayList<>();
```

Comment:
use a ClassNameConverter to convert between ClassName and Declarable. See `PdxConverter` as an example.

Reply:
Good idea. One concern a had about the ClassNameConverter logic is that it does not support all of the xml logic. For the parameters it only supports "string" values and not "declarable" values. Is it okay to just not handle the "declarable" parameter values?